src/uct/ib/mlx5/dc/dc_mlx5_devx.c

```diff
@@ -49,11 +49,12 @@ ucs_status_t uct_dc_mlx5_iface_devx_create_dct(uct_dc_mlx5_iface_t *iface)
                               ib_iface->config.max_inl_cqe[UCT_IB_DIR_RX], 1));
     UCT_IB_MLX5DV_SET(dctc, dctc, atomic_mode,
                       uct_ib_mlx5_get_atomic_mode(ib_iface));
-    if (uct_ib_iface_is_roce(&iface->super.super.super)) {
+    if (md->flags & UCT_IB_MLX5_MD_FLAG_DP_ORDERING_OOO_RW_DC) {
         UCT_IB_MLX5DV_SET(dctc, dctc, dp_ordering_0,
                           ucs_ternary_auto_value_is_yes_or_try(
                                   ib_iface->config.dp_ordering_ooo));
-        UCT_IB_MLX5DV_SET(dctc, dctc, dp_ordering_1, 0);
+        UCT_IB_MLX5DV_SET(dctc, dctc, dp_ordering_1,
```

Comment:
values supported for the two-bits fields `dp_ordering` are 0, 1 or 2, or is there are a later PRM version?

Used: Table 3731

Reply:
I don't know if there is a later PRM but we looked here: https://lore.kernel.org/netdev/f0e05a6d-6f9c-4351-af7a-7227fb3998be@nvidia.com/T/ 
and it seems only dp_ordering_1 is relevant for our case, and it also shows x4 BW in RC